semgrep-monitor

```diff
@@ -50,11 +75,40 @@ def generate_message(notify_reason: str) -> Dict[str, Any]:
     ]
 
 
-def main():
+def report_start_to_semgrep_app():
+    if not args.semgrep_app_token:
+        return
+
+    api_url = f"{args.semgrep_app_url}/api/agent/deployment/{args.deployment_id}/scan"
+    metadata = {
+        "repository": repo_name,
```

Comment:
You should add any other relevant metadata here as well; the backend has no schema expectation.

@dlukeomalley which metadata are you looking for?

Off the top of my head, I would think that an array of committers and authors would both be useful.